lockbox-ios/Presenter/FxAPresenter.swift

```diff
@@ -49,23 +51,27 @@ class FxAPresenter {
 
     init(view: FxAViewProtocol,
          dispatcher: Dispatcher = .shared,
-         accountStore: AccountStore = AccountStore.shared
+         accountStore: AccountStore = AccountStore.shared,
+         adjust: Adjust
```

Comment:
Similarly, is there not a good way to have an application-wide instance of `Adjust`?